Programming/bundle/nlparser/src/main/webapp/jsp/tabs.jsp

```diff
@@ -34,6 +34,7 @@
 
                     <input id="search-input-${domainEntry.key}" name="q" type="text" class="input-large search-input"  
                            placeholder="Type your question.." data-searchdomain="${domainEntry.key}"></input>
+                    <input type="submit" style="position: absolute; left: -9999px"/>
```

Comment:
display none should work since ie4? https://developer.mozilla.org/en-US/docs/Web/CSS/display#Browser_compatibility